addons/sourcemod/scripting/gokz-core/timer/pause.sp

```diff
@@ -65,31 +57,56 @@ bool CanPause(int client)
 		if (hasResumedInThisRun[client]
 			 && GetEngineTime() - lastResumeTime[client] < GOKZ_PAUSE_COOLDOWN)
 		{
-			GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Just Resumed)");
-			GOKZ_PlayErrorSound(client);
+			if (showError)
+			{
+				GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Just Resumed)");
+				GOKZ_PlayErrorSound(client);
+			}
 			return false;
 		}
 		else if (!Movement_GetOnGround(client)
 			 && !(Movement_GetSpeed(client) == 0 && Movement_GetVerticalVelocity(client) == 0))
 		{
-			GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Midair)");
-			GOKZ_PlayErrorSound(client);
+			if (showError)
+			{
+				GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Midair)");
+				GOKZ_PlayErrorSound(client);
+			}
 			return false;
 		}
 		else if (BhopTriggersJustTouched(client))
 		{
-			GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Just Landed)");
-			GOKZ_PlayErrorSound(client);
+			if (showError)
+			{
+				GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Just Landed)");
+				GOKZ_PlayErrorSound(client);
+			}
 			return false;
 		}
 		else if (AntiPauseTriggerIsTouched(client))
 		{
-			GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Anti Pause Area)");
-			GOKZ_PlayErrorSound(client);
+			if (showError)
+			{
+				GOKZ_PrintToChat(client, true, "%t", "Can't Pause (Anti Pause Area)");
+				GOKZ_PlayErrorSound(client);
+			}
 			return false;
 		}
 	}
 	
+	// Call Pre Forward
+	Action result;
+	Call_GOKZ_OnPause(client, result);
```

Comment:
Why would we move this? If I understand correctly, this would change the behavior of the forward, calling it on every pause check instead of every pause. I know that it's only called if you can pause, but you might check for the capability of pausing without actually pausing in the end.

Reply:
I don't remember why I moved this anymore, I'll revert it.